File: app/src/main/java/com/happs/ximand/ringcontrol/model/object/command/AddTrustedDeviceCommand.java
package com.happs.ximand.ringcontrol.model.object.command;

public class AddTrustedDeviceCommand extends BluetoothCommand<String> {

    private static final byte COMMAND_CODE = 1;

    protected AddTrustedDeviceCommand() {
        super(COMMAND_CODE);
    }

    @Override
    public byte[] getCommand() {
        return null;
    }

    @Override
    public String getMainContent() {
        throw new UnsupportedOperationException();
    }
}

File: app/src/main/java/com/happs/ximand/ringcontrol/model/object/command/AddTrustedDeviceCommand.kt
package com.happs.ximand.ringcontrol.model.`object`.command


class AddTrustedDeviceCommand : BluetoothCommand<String?>(COMMAND_CODE) {
    override fun toByteArray(): ByteArray? {
        return null
    }

    override fun getMainContent(): String {
        throw UnsupportedOperationException()
    }

    companion object {
        private const val COMMAND_CODE: Byte = 1
    }
}